pkg/instrumentation/sdk.go

```diff
@@ -502,6 +503,26 @@ func (i *sdkInjector) addParentResourceLabels(ctx context.Context, uid bool, ns
 			if uid {
 				resources[semconv.K8SJobUIDKey] = string(owner.UID)
 			}
+
```

Comment:
I think it would be really nice to add an E2E test for this situation

Reply:
e2e done, pls give it another review :-)